src/Metrics/WordErrorRate.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace ItkDev\SentenceSimilarityMetrics;
+
+class WordErrorRate
+{
+    /**
+     * Calculates word error rate (WER) between two sentences.
+     */
+    public function wer(string $reference, string $hypothesis): float
```

Comment:
Same here with doc-blocks for parameters?

Reply:
Added docs!